docs/index.md

```diff
@@ -37,3 +37,16 @@ After this, files in Music or Videos folders can be played:
 ```text
 $ mediaplayer-app Music/file.mp3
 ```
+
+## Configure gstreamer1.0 backend
+
+Some properties of the playbin object can be configured by creating a file in $HOME/.config/media-hub/media-hub.conf containing:
+```text
+# Playbin buffer-duration, in nano seconds, or 0 to disble
+export PLAYBIN_BUFFER_DURATION=5000000000
```

Comment:
This is not a good example, since the `buffer-size` only holds a 32 bit number.

Reply:
PLAYBIN_BUFFER_DURATION is used to set the 'buffer-duration' property. This property is specified in nanoseconds and it's type is gint64.